src/object/srv_obj.c

```diff
@@ -256,6 +256,11 @@ obj_bulk_transfer(crt_rpc_t *rpc, crt_bulk_op_t bulk_op, bool bulk_bind,
 	crt_bulk_perm_t		bulk_perm;
 	int			i, rc, *status, ret;
 
+	if (remote_bulks == NULL) {
+		D_ERROR("No remote bulks provided\n");
+		return -DER_INVAL;
+	}
```

Comment:
seems NULL remote_bulks is impossible? if so maybe just assert it.

Reply:
Ok, I'll update it if the patch needs be refreshed, otherwise, I'll make the change in follow-on SDL patch.